aiken-contracts/fractional/README.md

```diff
@@ -29,23 +29,23 @@ Use the `create_wallet.sh` script to auto generate simple wallets for testing.
 
 First create the reference scripts with `00_createScriptReferences.sh` found in the scripts folder.
 
-Next, we need to create the data reference UTxO using `01_createReferenceUTxO.sh` script inside the reference subfolder. This folder contains test scripts for updating data on the data UTxO. These scripts require a valid multisig using the keeper set of test wallets.
+Next, we need to create the data reference UTxO using `01_createReferenceUTxO.sh` script inside the `reference` subfolder. This folder contains test scripts for updating data on the data UTxO. These scripts require a valid multisig using the keeper set of test wallets.
```

Comment:
Maybe say something about how the reference utxo should be held in the reference test wallet and is identified by the NFT specified in the start json file?